lang/po/Firmware_no.po

```diff
@@ -2549,6 +2549,11 @@ msgstr "M600 filamentskifte. Sett inn en ny filament eller løs ut den gamle."
 msgid "Sensitivity"
 msgstr "Sensitivitet"
 
+#. MSG_MBL_FAILED_Z_CAL c=20 r=4
+#: ../../Firmware/Marlin_main.cpp:2976
+msgid "Mesh bed leveling failed. Please run Z calibration."
+msgstr "Sengeplanering feilet. Kjør Z-kalibrering."
```

Comment:
```suggestion
msgstr "Plateplanering feilet. Kjør Z-kalibrering."
```

Reply:
Not sure about this correction. Sengeplanering is used elsewhere in the file for mesh bed leveling.
"Plate" is used mainly to talk about the sheets, so I think Sengeplanering is more correct.
Should be consistent at least.

Reply:
okay lets stick to "seng" then to be consistent.

Reply:
Thanks guys, what is your verdict now. Both agree that it stays like I "translated"?
BTW: I looked up the PO file for similar text and reused it to be consistent.